sw/airborne/modules/gps/gps_ubx.c

```diff
@@ -104,6 +104,12 @@ void gps_ubx_init(void)
   gps_ubx.error_last = GPS_UBX_ERR_NONE;
 
   gps_ubx.state.comp_id = GPS_UBX_ID;
+  
+#if GPS_UBX_COLDSTART
```

Comment:
it should be documented in the module's documentation (xml file)

Reply:
Maybe add it as a define like this:
```
#ifndef GPS_UBX_BOOTRESET
#define GPS_UBX_BOOTRESET 0
#endif
...
 gps_ubx.reset = GPS_UBX_BOOTRESET;
```

Makes the code easier to read.